server/src/main/java/org/opensearch/index/query/SpanWithinQueryBuilder.java

```diff
@@ -197,4 +198,11 @@ protected boolean doEquals(SpanWithinQueryBuilder other) {
     public String getWriteableName() {
         return NAME;
     }
+
+    @Override
+    public void visit(QueryBuilderVisitor visitor) {
+        visitor.accept(this);
+        visitor.getChildVisitor(BooleanClause.Occur.SHOULD).accept(big);
+        visitor.getChildVisitor(BooleanClause.Occur.SHOULD).accept(little);
```

Comment:
I think these should both be `MUST`, like `SpanContainingQueryBuilder` -- since it sounds like they have similar matching criteria, but just iterate through positions differently (`SpanWithin` lets `little` do the driving, while `SpanContaining` advances with `big`).

Reply:
Gotcha